### PR TITLE
Set up basic Channels interface

### PR DESCRIPTION
**Pull request by Aelly-A**

### CODE REVIEW

File: backend/backend/asgi.py [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-
 import os
 
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter, URLRouter
+from channels.security.websocket import AllowedHostsOriginValidator
 from django.core.asgi import get_asgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'backend.settings')
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "mysite.settings")
```
Comment (goodCoderXD) [inline backend/backend/asgi.py:8]: leave that as backend.settings

File: backend/backend/asgi.py [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+# Initialize Django ASGI application early to ensure the AppRegistry
+# is populated before importing code that may import ORM models.
+django_asgi_app = get_asgi_application()
+
+import guineapig.routing
```
Comment (goodCoderXD) [inline backend/backend/asgi.py:13]: move imports to the top

File: backend/backend/settings.py [hunk 3]
```diff
@@ -122,3 +124,12 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+CHANNEL_LAYERS = {
```
Comment (goodCoderXD) [inline backend/backend/settings.py:128]: can you link where it says to do this? We don't have redis.

**Review by goodCoderXD — CHANGES_REQUESTED** [review]

**Review by goodCoderXD — APPROVED** [review]

**Comment by emijane** [comment]
LGTM!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Pull request merged**